modules/theme-tailwind-magento/cypress/support/pageObjects/account/login.js

```diff
@@ -0,0 +1,15 @@
+import getEmailInput from './getEmailInput'
+import getPasswordInput from './getPasswordInput'
+import getSignInButton from './getSignInButton'
+import getMicroAccount from './getMicroAccount'
+
+export default function login(params = {}) {
```

Comment:
Have you considered TypeScript? It could be helpful to have these input parameters typed, so that IDE can have better autocompletion.